Neos.Neos/Classes/AssetUsage/Projection/AssetUsageProjection.php

```diff
@@ -200,15 +199,14 @@ private function getAssetIdsByProperty(SerializedPropertyValues $propertyValues)
     }
 
     /**
-     * @param string $type
      * @param mixed $value
      * @return array<string>
      * @throws InvalidTypeException
      */
     private function extractAssetIds(string $type, mixed $value): array
     {
-        if ($type === 'string' || is_subclass_of($type, \Stringable::class)) {
-            preg_match_all('/asset:\/\/(?<assetId>[\w-]*)/i', (string)$value, $matches, PREG_SET_ORDER);
+        if (is_string($value)) {
```

Comment:
i dont understand this change really ...

Reply:
If type is instanceof Stringable but the value ist array (we're handling serialized property values here), it crashed before